packages/messageport/lib/messageport_tizen.dart

```diff
@@ -0,0 +1,163 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+final MethodChannel _channel = MethodChannel('tizen/messageport');
```

Comment:
Please resolve all linter warnings. You can check them either in the IDE or using the `flutter analyze` command.

![image](https://user-images.githubusercontent.com/40190588/119643151-61ab1900-be56-11eb-84bb-2bc5d9998992.png)

![image](https://user-images.githubusercontent.com/40190588/119643086-522bd000-be56-11eb-8a7b-3c120138ba05.png)